www/wip_new_website/content/index.md

```diff
@@ -72,6 +72,12 @@
 <script type="module" src="/wip/repl.js"></script>
 </section>
 
+## Examples
+
+**TODO MOVE ME TO A BETTER LOCATION**
+
+[EXAMPLES](/wip/examples)
+
```

Comment:
To be determined where these should be actually linked from, if there isn't room in the site nabber on mobiles. 

Maybe we can display in the navbar on larger screens, and hide on smaller mobile screens. It could also be linked from the front page.